tests/Tests/800__culler/culler.robot

```diff
@@ -105,8 +112,10 @@ Verify That "Stop Idle Notebook" Setting Is Not Overwritten After Restart Of Ope
     Oc Delete    kind=Pod     namespace=redhat-ods-operator    label_selector=name=rhods-operator
     sleep   5    msg=waiting time for the operator pod to be replaced with new one
```

Comment:
shouldn't the test wait until the operator pod is "running"? After 5 seconds, it will be in "init" state